lib/Models/Catalog/CatalogItems/ApiTableCatalogItem.ts

```diff
@@ -253,4 +253,28 @@ export class ApiTableCatalogItem extends AutoRefreshingMixin(
   }
 }
 
+/**
+ * Return the value at json path of the data object.
+ *
+ * This works exactly like the lodash.get() function but adds support for
+ * traversing array objects.  For eg, the lodash.get() does not support a path
+ * like: `a.users[].name`, but this function will correctly return a `{name}[]`
+ * array if they exist. The particular syntax for array traversal
+ * is borrowed from `jq` CLI tool.
+ */
```

Comment:
Can we define the type of `data` as an object where we dont know the properties maybe?